rmw_iceoryx_cpp/src/internal/iceoryx_name_conversion.cpp

```diff
@@ -85,14 +96,16 @@ get_name_n_type_from_service_description(
     service_lowercase + ARA_DELIMITER + event);
 }
 
-std::tuple<std::string, std::string, std::string> get_service_description_from_name_n_type(
+std::tuple<std::string, std::string, std::string>
+get_service_description_from_name_n_type(
   const std::string & topic_name,
   const std::string & type_name)
 {
   auto position_ara_delimiter = type_name.find(ARA_DELIMITER);
 
   if (position_ara_delimiter == std::string::npos) {
     // ROS 2.0 Naming
+    // TODO(karsten1987): Filter out iceoryx Introspection topics
```

Comment:
@michael-poehnl Do we have to filter the introspection topics on the way out as well?